lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -25,7 +25,9 @@ def receive(event)
 
     # Receive an array of events and immediately attempt to index them (no buffering)
     def multi_receive(events)
-      retrying_submit(events.map {|e| event_action_tuple(e) })
+      events.each_slice(@flush_size) do |slice|
```

Comment:
Can you add a comment here, saying why we are doing this in relationship to `batch_size`?